account_financial_report_webkit/report/common_partner_balance_reports.py

```diff
@@ -259,7 +261,7 @@ def compute_partner_balance_data(self, data, filter_report_type=None):
         # get details for each accounts, total of debit / credit / balance
         accounts_by_ids = self._get_account_details(
             account_ids, target_move, fiscalyear, main_filter, start, stop,
-            initial_balance_mode)
+            initial_balance_mode, context=lang_ctx)
```

Comment:
Why not use `data['form']['used_context']` (or a similar form) instead of this limited context which only contains `lang` ?
`data['form']['used_context']` is defined here : https://github.com/odoo/odoo/blob/8.0/addons/account/wizard/account_report_common.py#L183 and `lang` key is already defined.